tests/functional/ironicneutronagent_controller_test.go

```diff
@@ -119,7 +119,7 @@ var _ = Describe("IronicNeutronAgent controller", func() {
 				ironicNames.INAName,
 				ConditionGetterFunc(INAConditionGetter),
 				condition.ReadyCondition,
-				corev1.ConditionUnknown,
+				corev1.ConditionFalse,
```

Comment:
If you do another revision change string in L117 and L153 to "is false Ready"

Reply:
Yes. That totally makes sense. I've fixed this (and the line below)